### DISCUSSION TITLE
Getting Error while setup with Docker Image in Development phase

### DISCUSSION THREAD

**Discussion by nikhil32002** [DefectDojo/django-DefectDojo#6145] [Q&A]
![Docker_issue](https://user-images.githubusercontent.com/69077656/162599638-beff7577-e135-47fe-b651-43dcc0b88b44.jpeg)

Hello Everyone,

I am trying to set up a defect-dojo on my local machine(windows), I am referring to the documentation provided by the team, but for the below command, I am getting the error-

**docker/setEnv.sh dev
./dc-build.sh
./dc-up.sh**

Error-   **docker: '/setEnv.sh' is not a docker command.
See 'docker --help'**

Please Help, Thanks in Advance!!

**Comment by kramdass08** [comment]
Instead of Command Prompt. Use Windows PowerShell.

**Comment by kramdass08** [comment]
Before running the install scripts, Install Docker for Windows and Docker Compose.
